yarn.lock

```diff
@@ -9556,9 +9556,9 @@ prettier-linter-helpers@^1.0.0:
     fast-diff "^1.1.2"
 
 prettier@^1.18.2, prettier@^1.7.0:
-  version "1.18.2"
-  resolved "https://registry.yarnpkg.com/prettier/-/prettier-1.18.2.tgz#6823e7c5900017b4bd3acf46fe9ac4b4d7bda9ea"
-  integrity sha512-OeHeMc0JhFE9idD4ZdtNibzY0+TPHSpSSb9h8FqtP+YnoZZ1sl8Vc9b1sasjfymH3SonAF4QcA2+mzHPhMvIiw==
+  version "1.19.1"
```

Comment:
> Looks like this requires quite a few formatting changes, and I'm not
> sure I like all of them -- in test files, especially, it's nice when
> we can fit something simple onto a single line. Ah, well.

Can we change the settings to not require the changes that we don't like? I'd prefer to just drop this commit from this PR, so we can talk about formatting changes separately from the rest of these updates.

Reply:
Sure. Prettier isn't very configurable, and my hopes aren't very high that we'll be able to get what we want in Prettier's config. Possibly there's something we could do in `tools/formatting.eslintrc.yaml`, which (with prettier-eslint-cli) we use to overrule some of Prettier's opinions by running ESLint with some targeted rules.

For strategy on removing the commit from this PR (while keeping the headline big-upgrade commit):

I'm wary of making the big upgrade commit more complicated; it would be great not to have to change the command there from `yarn upgrade` to something like `yarn upgrade $(tools/deps | grep -vx -e prettier)` and explain what `prettier` is doing in the command there.

One option would be to have a commit that pins `prettier` to its current version (using a range without a caret), and then `yarn upgrade` would see that and not try to upgrade it.

Reply:
> One option would be to have a commit that pins `prettier` to its current version (using a range without a caret), and then `yarn upgrade` would see that and not try to upgrade it. Then after this PR is in, we can think about upgrading `prettier` and adding a caret back.

Yeah, that'd be a fine alternative to the `$(tools/deps | grep -v …)` thing.

Reply:
Also note that we can always disable Prettier locally with `// prettier-ignore` -- that applies to just the immediately following statement or expression, so it's generally pretty finely targeted. We have a few dozen of those already, and I think it's appropriate to have a fairly low threshold for applying it when Prettier tries to do something ugly.